README.md

```diff
@@ -1,3 +1,22 @@
 # openmrs-esm-root-config
 
-A javascript module for setting up and configuring your SPA.
+`@openmrs/root-config`
+
+A javascript module for setting up and configuring your SPA. See [openmrs-rfc-frontend](https://github.com/openmrs/openmrs-rfc-frontend)
+for what this project is and how it works.
+
+## Local development
+
+```sh
+npm install
+npm start <port-number> # e.g., npm start 8080
```

Comment:
would this be 
> npm start -- --port <port numer>

update document for HTTPs as well?
> npm start -- --https --port <port number>
> npm start -- --https --key ./path-to-key --cert ./path-to-public --port <port number>

Reply:
see https://github.com/openmrs/openmrs-esm-devtools/pull/3.

Also, the documentation will all be moving to the wiki this week.